extensions/amp-social-share/0.2/social-share.css.js

```diff
@@ -54,3 +54,6 @@ export const SMS = {'backgroundColor': 'ca2b63'};
 
 /* "system" styling */
 export const SYSTEM = {'backgroundColor': '000000'};
+
+/* "default" styling */
+export const DEFAULT = {'backgroundColor': 'd3d3d3'};
```

Comment:
Why is this needed?

Reply:
The default color is currently white, which is invisible in most cases against a white background (including in storybook).  So I felt it made sense to explicitly set some visible color.